lib/auth.coffee

```diff
@@ -23,6 +23,15 @@ getAuth = (deps, opts) ->
 	twoFactor = require('./2fa')(deps, opts)
 	exports = {}
 
+	normalizeError = (err) ->		#todo
+		# console.log(err.code)
+		if err.statusCode == 401
+			return new errors.ResinNotLoggedIn()
+		else if err.code == 'ResinMalformedToken' and not savedToken?
```

Comment:
`savedToken` isn't defined in this function. I don't think you need that check anyway - if we get a malformed token error somehow, then I think we know we're definitely not logged in.